lib/validators/approvals.js

```diff
@@ -98,11 +99,14 @@ class Approvals extends Validator {
       }
 
       teamMembers = _.union(teamMembers, owners)
+      let validatorContext = { name: 'approvals' }
 
+      output.push(constructOutput(validatorContext, _.union(teams, owners), limitOption, {status: 'info', description: `Only the following teams and users approval will counted`}))
```

Comment:
You can use "Only approvals from following sources are counted." verbiage instead
This will cover all future options in `limit` as well

Reply:
Should we use `_.union(teams, owners)` or `approvedReviewers` ?

Reply:
i thought input should be `sources` rather than the current `approvedReviewers` that is counted

Reply:
do you think `approvedReviewers` make more sense here??

Reply:
Yeah, because `input` is already displayed in `settings` field in the output